### PR TITLE
ome_alert_policies_message_id_info-module

### PR DESCRIPTION
**Pull request by ShivamSh3**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### OUTPUT

```paste below

```
##### ADDITIONAL INFORMATION

```paste below

```
# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

### CODE REVIEW

**Comment by ansible-collections-svc** [comment]
Can one of the admins verify this patch?

File: plugins/modules/ome_alert_policies_message_id_info.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+from ansible_collections.dellemc.openmanage.plugins.module_utils.ome import RestOME, ome_auth_params
+from ansible_collections.dellemc.openmanage.plugins.module_utils.utils import remove_key
+from ansible.module_utils.six.moves.urllib.error import URLError, HTTPError
+from ansible.module_utils.urls import ConnectionError, SSLValidationError
+
+ALERT_MESSAGE_URI = "AlertService/AlertMessageDefinitions"
+
+
+def main():
+    specs = {}
```
Comment (ABHISHEK-SINHA10) [inline plugins/modules/ome_alert_policies_message_id_info.py]: line 115 and 116 can be merged, if it doesn't have input, simply assign to specs, no need to update empty dictionary

File: plugins/modules/ome_alert_policies_message_id_info.py [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+description:
+   - "This module retrieves the message ID information of alert policies for OpenManage Enterprise."
+extends_documentation_fragment:
+  - dellemc.openmanage.ome_auth_options
+requirements:
+    - "python >= 3.9.6"
+author: "Shivam Sharma (@ShivamSh3)"
+notes:
+    - Run this module from a system that has direct access to Dell OpenManage Enterprise.
+    - This module supports IPv4 and IPv6 addresses.
```
Comment (ABHISHEK-SINHA10) [inline plugins/modules/ome_alert_policies_message_id_info.py:32]: If this module supports check_mode, add the notes, as I can see same in line 119

**Review by jagadeeshnv — DISMISSED** [review]

File: plugins/modules/ome_alert_policies_message_id_info.py (hunk 1 above)
Comment (ShivamSh3) [reply plugins/modules/ome_alert_policies_message_id_info.py]: updated.

File: plugins/modules/ome_alert_policies_message_id_info.py (hunk 2 above)
Comment (ShivamSh3) [reply plugins/modules/ome_alert_policies_message_id_info.py:32]: updated.

**Review by ABHISHEK-SINHA10 — DISMISSED** [review]

File: plugins/modules/ome_alert_policies_message_id_info.py [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+  type: str
+  description: Error description in case of error.
+  returned: on error
+  sample: "HTTP Error 501: 501"
+message_ids:
+  type: dict
+  description: Details of the message ids.
+  returned: success
+  sample: [
+    {
```
Comment (felixs88) [inline plugins/modules/ome_alert_policies_message_id_info.py]: keep it two spaces instead of four

**Review by felixs88 — DISMISSED** [review]

File: plugins/modules/ome_alert_policies_message_id_info.py (hunk 3 above)
Comment (ShivamSh3) [reply plugins/modules/ome_alert_policies_message_id_info.py]: removed spaces

**Review by felixs88 — DISMISSED** [review]

**Review by ABHISHEK-SINHA10 — DISMISSED** [review]

**Comment by ShivamSh3** [comment]
AV

**Comment by ShivamSh3** [comment]
Checkmarx

File: playbooks/ome/alert_policy/ome_alert_policies_message_id_info.yml
Comment (ABHISHEK-SINHA10) [inline playbooks/ome/alert_policy/ome_alert_policies_message_id_info.yml:1]: remove connection: local and add delegate_to: localhost to each task

**Review by ABHISHEK-SINHA10 — DISMISSED** [review]

**Review by felixs88 — DISMISSED** [review]

**Review by sachin-apa — DISMISSED** [review]

**Review by sachin-apa — DISMISSED** [review]

File: plugins/modules/ome_alert_policies_message_id_info.py [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: ome_alert_policies_message_id_info
+short_description: Get message ID information of alert policies.
+version_added: "8.2.0"
+description:
+   - "This module retrieves the message ID information of alert policies for OpenManage Enterprise
```
Comment (ABHISHEK-SINHA10) [inline plugins/modules/ome_alert_policies_message_id_info.py:21]: Add description of what this module does and generate rst, check other modules for reference.

Comment (ABHISHEK-SINHA10) [reply plugins/modules/ome_alert_policies_message_id_info.py:21]: resolving as it's mentioned.

**Review by ABHISHEK-SINHA10 — DISMISSED** [review]

File: plugins/modules/ome_alert_policies_message_id_info.py [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+    hostname: "192.168.0.1"
+    username: "username"
+    password: "password"
+    ca_path: "/path/to/ca_cert.pem"
+'''
+
+RETURN = r'''
+---
+msg:
+  description: Successfully retrieved alert policies message ids information.
```
Comment (ABHISHEK-SINHA10) [inline plugins/modules/ome_alert_policies_message_id_info.py]: Description is not correct for msg. Please update.

Comment (ShivamSh3) [reply plugins/modules/ome_alert_policies_message_id_info.py]: updated

**Review by ABHISHEK-SINHA10 — DISMISSED** [review]

**Review by Kritika-Bhateja-03 — DISMISSED** [review]

**Review by ABHISHEK-SINHA10 — DISMISSED** [review]

**Review by Kritika-Bhateja-03 — DISMISSED** [review]

**Review by jagadeeshnv — APPROVED** [review]

**Review by ABHISHEK-SINHA10 — APPROVED** [review]

**Review by rajshekarp87 — APPROVED** [review]

**Pull request merged**